python/lsst/ts/salobj/domain.py

```diff
@@ -32,9 +33,11 @@
 from lsst.ts import idl
 
 # Length of DDS read queue
-# I would prefer to set this in the QoS file,
+# Warning: this must be equal to or longer than the queue length in the
+# QoS XML file.
+# I would prefer to set this constant *from* the QoS XML file,
 # but dds does not make the information available
```

Comment:
I'll change it to:
```
# This information is not available from dds objects, so I set queue depth
# instead of using the value specified in the QoS XML file.
```
I could actually read the QoS XML file, but then I'd have to rely on the queue depth being specified there (if it is omitted then OpenSplice presumably has a default value) and it require a lot of code. The lack of introspection for OpenSplice dds objects is maddening.